.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 cache: bundler
 services:
 - redis-server
+before_install:
+- sudo pip install yamllint
```

Comment:
Is this the same as the `yamllint` gem?

Reply:
From the description of the yamllint gem:

```
Checks YAML files for correct syntax. Currently it checks for:

    Valid YAML syntax
    Overlapping key definitions in YAML files, where the last definition would win
```

It would be much better if we could use a rubygem, and I had a lot of hesitancy introducing a python package into the bot! But I haven't seen a better solution - it does a lot more than the gem as it also checks indentation, trailing whitespace, whitespace consistency etc...